### PR TITLE
mention new Selsis et al. results

### PR DESCRIPTION
**Pull request by matiscke**

### CODE REVIEW

File: src/tex/ms.tex [hunk 1]
```diff
@@ -553,8 +553,9 @@ \subsubsection{Key factors influencing tests of the runaway greenhouse hypothesi
...
-However, already about $\SIrange{\sim 10}{20} \, \mathrm{bar}$ of water vapor -- corresponding to a minor fraction of one Earth ocean and thus the lower limit of water on Earth -- is enough to sustain sufficiently high surface temperatures to keep the planet in a magma ocean stage~\citep{Boukrouche2021,Lichtenberg2021c}.
-The fraction of planets fulfilling these requirements has an impact on the amplitude of the demographic imprint of the runaway greenhouse transition.
+\revii{It was shown that} already about $\SIrange{\sim 10}{20} \, \mathrm{bar}$ of water vapor -- corresponding to a minor fraction of one Earth ocean and thus the lower limit of water on Earth -- is enough to sustain sufficiently high surface temperatures to keep the planet in a magma ocean stage~\citep{Boukrouche2021,Lichtenberg2021c}.
+\revii{However, recent simulations accounting for radiative–convective profiles suggest the existence of cooler runaway greenhouse states that do not necessarily yield a surface magma ocean~\citep{Selsis2023}.
```
Comment (timlichtenberg) [inline src/tex/ms.tex:557]: Suggestion for a few minor additions in the last sentence:
"However, recent simulations accounting for radiative–convective profiles *of near-surface atmospheric layers* suggest the existence of cooler runaway greenhouse states that do not necessarily yield a *molten surface*~\citep{Selsis2023}."

**Review by timlichtenberg — APPROVED** [review]
A few minor additions to one sentence, directly added to ms.tex

File: src/tex/ms.tex (hunk 1 above)
Comment (matiscke) [reply src/tex/ms.tex]: Agreed, thank you!

**Comment by matiscke** [comment]
Additions finished, reviewed, and de-bolded. Merging this into `main`.

**Comment by matiscke** [comment]
Additions finished, reviewed, and de-bolded. Merging this into `main`.

**Pull request merged**